### ISSUE TITLE
i have some questions about rotation

### ISSUE THREAD

**Issue by cumthxy** [Delgan/loguru#1032]
i have some questions about rotation  

i use this code `logger.add("call_logs.log", rotation="10s")`
when i manually run the code over and over again.  
i find some logs logs appear in the same file。
<img width="592" alt="image" src="https://github.com/Delgan/loguru/assets/22782047/8b478b5e-aca7-4b67-aee0-48cbb33e557c">

how find this issue?

**Comment by Delgan** [comment]
Hi @cumthxy.

Does your code use multi-processing or multi-threading? Are you sure you're not inadvertently calling `logger.add("file.log")` multiple times?

It's not unusual to encounter file-logging issues due to misconfiguration of the logger.

**Comment by cumthxy** [comment]
> Hi @cumthxy.
> 
> Does your code use multi-processing or multi-threading? Are you sure you're not inadvertently calling `logger.add("file.log")` multiple times?
> 
> It's not unusual to encounter file-logging issues due to misconfiguration of the logger.
> 
> If you could detail your usage or even provide a minimum reproducible example, it would help me understand the problem.

yes ,i have four python script need run at same time,and need log in same file

i write code like `this: logger.add("call_logs.log", rotation="10s")` in util.py 
and in other python script `from util import logger`.
 this calling logger.add("file.log") multiple times.

how to sovle this?

**Comment by Delgan** [comment]
You should only configure the logger once. In your main file, you can add your handler using `logger.add("file.log")`:

```python
# main.py
from loguru import logger
import other

if __name__ == "__main__":
    logger.add("file.log", rotation="10s")
    logger.info("Starting script")
    other.some_function()
```

In your other modules, you can just import the `logger` from `loguru`. Because the `logger` is a global instance, it will inherit the handlers that were previously configured.

```python
# other.py
from loguru import logger

def some_function():
    logger.info("Message")  # This will be logged to "file.log" automatically
```
